### PR TITLE
add chat-app-static landing page and assets

### PR DESCRIPTION
**Pull request by nullsmegh**

### CODE REVIEW

File: chat-app-static/styles/chat-app.css [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+    font-size: 12px;
+    line-height: normal;
+    color: #51DB98;
+    display: block;
+}
+
+.chat-container {
+    border-left: solid 1px #EEEEEE;
+    flex-grow: 1;
+    height: calc(100vh - 70px);
```
Comment (akshayjj1997) [inline chat-app-static/styles/chat-app.css]: why the background-color: #F5F5F5 for the entire container is not there (same as figma)?

File: chat-app-static/styles/chat-app.css [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    line-height: normal;
+}
+
+.chat-item-excerpt {
+    display: block;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: normal;
+    color: rgba(20, 20, 20, 0.64);
+    max-width: 220px;
```
Comment (akshayjj1997) [inline chat-app-static/styles/chat-app.css]: should there be a little bit of distance (margin-top) between the header (Noorain Panjwani) and excerpt (Lorem ispum bien coma sav que ore bie...) as in figma?

File: chat-app-static/styles/chat-app.css [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+    margin: 0;
+}
+
+.chat-user-status {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: normal;
+    color: #51DB98;
+    display: block;
```
Comment (akshayjj1997) [inline chat-app-static/styles/chat-app.css]: should there be a little bit of distance (margin-top) between the header (Megh Gandhi) and user status (Active now) as in figma?

File: chat-app-static/styles/chat-app.css (hunk 1 above)
Comment (nullsmegh) [reply chat-app-static/styles/chat-app.css]: It is given to the chat div. Not entirely sure why you would want to set the background to #F5F5F5 of the whole container.

File: chat-app-static/styles/chat-app.css (hunk 2 above)
Comment (nullsmegh) [reply chat-app-static/styles/chat-app.css]: > 1. The graphics and icons are not visible anywhere on windows, is it the same issue on other platforms?

You will have to serve the .html files from a server with root as the chat-app-static directory.

File: chat-app-static/styles/chat-app.css (hunk 1 above)
Comment (akshayjj1997) [reply chat-app-static/styles/chat-app.css]: I want class="chat" 's background to be applied the entire div (bg) of no-chat and empty-chat html pages as in figma

**Pull request merged**